wlcs_pinnacle/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "wlcs_pinnacle"
+version = "0.0.1"
+authors.workspace = true
+edition.workspace = true
+repository.workspace = true
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+smithay = { workspace = true }
+pinnacle = { path = "..", features = [ "wlcs" ] }
+pinnacle-api = { path = "../api/rust" }
+wayland-sys = { version = "0.31.1", features = ["client", "server"] }
+wlcs = "0.1"
+
+tracing = "0.1.40"
+tracing-subscriber = { version = "0.3.18", features = ["env-filter", "registry"] }
```

Comment:
We could probably make these workspace dependencies

Reply:
Done